src/main/java/co/inventorsoft/academy/springBootTask/service/impl/TeamServiceImpl.java

```diff
@@ -0,0 +1,74 @@
+package co.inventorsoft.academy.springBootTask.service.impl;
+
+import co.inventorsoft.academy.springBootTask.domain.dto.TeamDto;
+import co.inventorsoft.academy.springBootTask.domain.entity.Team;
+import co.inventorsoft.academy.springBootTask.domain.mapper.TeamMapper;
+import co.inventorsoft.academy.springBootTask.repository.TeamRepository;
+import co.inventorsoft.academy.springBootTask.service.TeamService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Scanner;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class TeamServiceImpl implements TeamService {
+
+    public static final int MINIMUM_TEAMS_NUMBER = 4;
+
+    private final TeamRepository teamRepository;
+
+    @Override
+    public void addTeams() {
+        int teamNumber = 0;
+        Scanner scanner = new Scanner(System.in);
+        boolean isActiveScanner = true;
+        while (isActiveScanner) {
+            System.out.println("Amount of teams: " + teamNumber);
+            System.out.println("Enter team (name, capitan, coach)");
+            System.out.println("Enter 'stop' to finish adding teams");
+            String currentLine = scanner.nextLine();
+            if (currentLine.equals("stop")) {
```

Comment:
Always compare static values versus dynamic.
Because of you have `currentLine` as `null` you will get `NullPointerException`.